pkg/operator/ceph/file/mds/mdsLivenessProbeTestsFiles/0FS-2MDS.json

```diff
@@ -0,0 +1,114 @@
+{
```

Comment:
Let's name the folder `test` instead of `mdsLivenessProbeTestsFiles`, to keep the directory structure simple.